lib/partition_setup.pm

```diff
@@ -38,6 +39,10 @@ our %partition_roles = qw(
   raw alt-a
 );
 
+sub is_new_storage_ng {
```

Comment:
I guess it serves same purpose as `is_storage_ng_newui` I've introduced [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/partition_setup.pm#L119), feel free to adjust it as I have included TW, as it was not affected for lvm_resize test suite.

Reply:
Also, same as below, I proposed this to @jknphy, but he used local variable for the perl module: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning_raid.pm#L21 We rather should use single method and not 3 different approaches. Again, feel free to leave that change to me, but please, let's use single method for detecting new UI at least.